examples/sfml/mesh.rs

```diff
@@ -0,0 +1,47 @@
+use sfml::{
+    graphics::{Color, Drawable, FloatRect, PrimitiveType, Vertex},
+    system::Vector2f,
+};
+
+pub struct VertexMesh(Vec<Vertex>);
+
+impl VertexMesh {
+    /// Create a new mesh with capacity for the given amount of vertices.
+    pub fn with_capacity(vertices: usize) -> Self {
```

Comment:
Maybe it makes sense to rename this struct to `QuadsMesh`, given that it can only be used to render quads? Then it would take `quads` argument and the `* 4` for the capacity can be moved in here.